sdk/core/azure-core/CHANGELOG.md

```diff
@@ -30,6 +30,9 @@
 - Changed the `Azure::Core::Http::HttpMethod` regular enum into an extensible enum class and removed the `HttpMethodToString()` helper method.
 - Removed `Azure::Core::Http::Request::GetHeadersAsString()`.
 - Introduced `Azure::Core::Context::Key` class which takes place of `std::string` used for `Azure::Core::Context` keys previously.
+- Renamed type `Azure::Core::Http::CurlTransportSSLOptions` to `Azure::Core::Http::CurlTransportSslOptions`.
+- Renamed member `Azure::Core::Http::CurlTransportOptions::SSLOptions` to `Azure::Core::Http::CurlTransportOptions::SslOptions`.
+- Renamed member `Azure::Core::Http::CurlTransportOptions::SSLVerifyPeer` to `Azure::Core::Http::CurlTransportOptions::SslVerifyPeer`.
```

Comment:
nit: To make it less noisy, we should avoid repeating the entire namespace multiple times.
```suggestion
- Renamed type `Azure::Core::Http::CurlTransportSSLOptions` to `CurlTransportSslOptions`.
- Renamed member `Azure::Core::Http::CurlTransportOptions::SSLOptions` to `SslOptions`.
- Renamed member `Azure::Core::Http::CurlTransportOptions::SSLVerifyPeer` to `SslVerifyPeer`.
```